service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -59,6 +66,19 @@ public void unfavoriteSite(JWTData userData, int siteId) {
         .and(FAVORITE_SITES.SITE_ID.eq(siteId));
   }
 
+  @Override
+  public FavoriteSitesResponse getFavoriteSites(JWTData userData) {
+    List<FavoriteSitesRecord> favoriteSites =
+        db.selectFrom(FAVORITE_SITES)
+            .where(FAVORITE_SITES.USER_ID.eq(userData.getUserId()))
+            .fetch();
+    List<Integer> favorites = new ArrayList<>();
+
+    favoriteSites.forEach(site -> favorites.add(site.getSiteId()));
```

Comment:
Here you can probably do something like 
```java
List<Integer> favorites = db.select(FAVORITE_SITES.SITE_ID)
    .from(FAVORITE_SITES).where(...).fetch();
```
or
```java
List<Integer> favoriteSites =
        db.selectFrom(FAVORITE_SITES)
            .where(FAVORITE_SITES.USER_ID.eq(userData.getUserId()))
            .fetch(FAVORITE_SITES.SITE_ID);
```
to remove the extra logic that loops through each record

Reply:
Ah good call, was just copying other code without thinking